web/src/components/tasks/LicenseStatusReceipt.tsx

```diff
@@ -114,11 +111,11 @@ export const LicenseStatusReceipt = (props: Props): ReactElement => {
 
   const getIcon = (status: LicenseStatus): string => {
```

Comment:
What is this section related to? Is it the Permit section, if so, it should be grouped together